lib/osf-components/addon/components/collection-submission-card/template.hbs

```diff
@@ -1,25 +1,44 @@
 <div local-class='page-container'>
-    <div local-class='icon-container'>
-        <FaIcon @icon='clock' @prefix='far' />
+    <div local-class='icon-container {{this.iconClass}}'>
+        <FaIcon data-test-submission-card-icon @icon={{this.icon}} @size='2x'/>
```

Comment:
Since this is a list card component, would your data test selectors be easier to use if you did `data-test-blah-blah={{this.project.id}}` or similar? If you've already done your tests and they're fine using a more complex test selector, or you only have tests on one card at a time, that's probably okay, unless it would make QA's life easier for selenium tests to do it this way.